_news-and-blog/2022-08-15-a-quick-guide-to-inclusive-design.md

```diff
@@ -0,0 +1,221 @@
+
+Mobile design is essential for equity, as [research shows](https://www.pewresearch.org/internet/fact-sheet/mobile/) about 15 percent of American adults depend on smartphones to go online.
+
+Reliance on smartphones is more common for younger adults, people who are lower-income, and lower education levels.
+
+Mobile design principles have a lot of crossover benefits that help people more easily use your site regardless of their
+abilities or screen size. And, generally, designing mobile-first will help ensure your content and designs don’t become
+overly complex.
+
+#### Here are some of the ways we use mobile-first to further support other inclusive design practices
+
+- Use enough space around clickable elements (e.g., buttons, links, form fields). On mobile, this prevents fingers and
+  thumbs from accidentally selecting the wrong thing. On desktop this makes it easier for people who have difficulty
+  using a mouse.
+- Write your content in the context of a small screen. This helps ensure your content is concise and gets to the point
+  quickly. (Tip: if drafting content in a text-based computer application like Microsoft Word, you can simulate a small
+  screen by increasing your side margins so you are only using three inches of space)
+- When sharing drafts or mock-ups for feedback internally, share them in the context of a small screen. This helps
+  counteract impulses by stakeholders, lawyers, and subject matter experts to add tons more content because when you
+  show them their longer content on a small screen, the low usability of this becomes unavoidable.
+- Remember that internal users benefit from mobile responsive tools too because they may shrink multiple windows down to fit on their screen side-by-side.
+
+![Squares made of dark blue, medium blue, gold and white triangles are piled on top of each other to form a pyramid of squares. This represents the USDS value of "hire and empower great people."](2022-08-15-a-quick-guide-to-inclusive-design-img/rVj8FA.png)
+
+### 5. Hire and empower great people
+
+Another one of the [six USDS values](/mission#our-values) is to “hire and empower great people.”
+
+As part of this, we strive to make sure that our own teams reflect the great diversity of the populations we’re serving. And once they’re in place, they are fully empowered and encouraged to do what’s needed to make sure our products are inclusive, easy-to-use, and equitable.
+
+When a team doesn’t have much diversity, it’s important to ask, “Who are we missing? Whose voices do we need to make
+sure to hear?” This helps us learn [how to recognize exclusion](https://uxpajournal.org/learning-recognize-exclusion/)
+and develop more inclusive practices in our hiring and design process.
+
+**It’s always nice to end with a quote, so here’s one from a user research participant to one of our designers:**
+
+_“As a disabled Veteran it means \_a lot_ to me that this type of site is being built up for us! It already appears to be
```

Comment:
Pull quote is not formatting if it's supposed to match pull quote style on project pages.

Reply:
This was just in the body of the text in the OG post, not a pull quote. It looks like it has some extra trailing characters on it which I'll fix!